C/CDCS/Compat.toml

```diff
@@ -1,9 +1,15 @@
 [0]
-MATLAB = "0.7.3-*"
 julia = "1"
 
+["0-0.1.0"]
+MATLAB = "0.7.3-*"
```

Comment:
This should probably be `0.7.3-0.7`?

Reply:
I agree, but this was added by an older release. I cannot make JuliaRegistrator change that through the Project.toml.

Reply:
It is possible to just edit the file directly in the github interface.

Reply:
#10240